website_lazy_load_image/models/ir_ui_view.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from lxml import etree
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    DEFAULT_SRC = 'data:image/gif;base64,R0lGODlhAQABAIAAAP///' \
```

Comment:
please rename to LAZYLOAD_DEFAULT_SRC and say what kind of image is and how is used in a comment ;)